src/core/public/chrome/ui/header/header_breadcrumbs.tsx

```diff
@@ -42,7 +42,18 @@ interface Props {
 
 export function HeaderBreadcrumbs({ appTitle$, breadcrumbs$ }: Props) {
   const appTitle = useObservable(appTitle$, 'OpenSearch Dashboards');
-  const breadcrumbs = useObservable(breadcrumbs$, []);
+  // const breadcrumbs = useObservable(breadcrumbs$, []);
+  const [breadcrumbs, setBreadcrumbs] = useState<ChromeBreadcrumb[]>([]);
+
+  useEffect(() => {
+    const subscription = breadcrumbs$.subscribe((newBreadcrumbs: ChromeBreadcrumb[]) => {
+      setBreadcrumbs(newBreadcrumbs);
+    });
+    return () => {
+      subscription.unsubscribe();
+    };
+  });
```

Comment:
Seems missing dependencies array
```suggestion
  }, []);
```

Reply:
It's by intention, if we add dependency here, it will be same as `useObservable` and that will not able capture `breadcrumbs$` change for all time instead it just capture the value for first time.

Reply:
this logic has been moved to workspace plugin